Saucier720-app/src/app/recipes/FEC/recipe-card/recipe-card.component.scss

```diff
@@ -119,23 +119,56 @@
   font-size: 16px;
 }
 
+.delete-icon,
 .favorite-icon {
   position: absolute;
   top: 10px;
   left: 10px;
   width: 20px; /* Adjust the width as needed */
   height: 20px; /* Adjust the height as needed */
-  opacity: 0.5; /* Set the initial opacity for the transparent effect */
+  opacity: 1; /* Set the initial opacity for the transparent effect */
   cursor: pointer;
   transition: opacity 0.3s ease-in-out; /* Add a smooth transition effect */
 }
 
+.favorite-icon {
+  margin-top: 0.485px; // added to make them look more even
+}
+
+.delete-icon {
+  margin-left: 25px; /* Add some spacing between the icons */
+  width: 22px; /* Adjust the width as needed */
+  height: 22px; /* Adjust the height as needed */
+  opacity: 0.8;
+  transition: opacity 0.3s ease-in-out; /* Add a smooth transition effect */
```

Comment:
Do you think instead of button it would be nicer to have like a bar go around the outside border that fills up? Not a prob with this one just could be UI issue to update

Reply:
yes that would be sick, seems like it might be difficult, but could be cool